org/mozilla/jss/pkcs11/attrs/CKAClass.java

```diff
@@ -0,0 +1,123 @@
+package org.mozilla.jss.pkcs11.attrs;
+
+import org.mozilla.jss.pkcs11.PKCS11Constants;
+
+/**
+ * CKAClass is an instance of a PKCS#11 CK_ATTRIBUTE with type = CKA_CLASS.
+ */
+public class CKAClass extends CKAttribute {
+    private long value;
+
+    public CKAClass() {
```

Comment:
One more comment :) This constructor allows creating `CKAClass` without initializing the value. If this is not allowed probably this constructor should be removed as well. Same thing with other subclasses of `CKAAttribute`.

Reply:
Should we drop it or make it private? I suppose it is better to have the one argument constructor rather than a no-argument constructor... And should I drop the `setValue(...)` thingies then? Currently it is using the no-argument constructor in all of the subclasses and calling `setvalue(...)`, though I could switch that to calling `super(...)` which'd probably be cleaner... I'll do that.